### PR TITLE
[dashboard] CHE_510 Adopt new style

### PR DESCRIPTION
**Pull request by benoitf**
**_3 Upvotes**_

### CODE REVIEW

**Comment by ashumilova** [comment]
+1

File: dashboard/src/components/widget/button-dropdown/che-button-dropdown.styl [hunk 1]
```diff
@@ -1,10 +1,10 @@
 che-button-dropdown .md-button.md-accent.md-raised.md-hue-2
   che-border-radius(5px)
   box-shadow-simple()
-  font-size 1em
+  font-size 1.6em
   font-weight bold
   width auto
-  line-height 30px
+  line-height 34Mpx
```
Comment (akurinnoy) [inline dashboard/src/components/widget/button-dropdown/che-button-dropdown.styl]: it seems a typo

Comment (benoitf) [inline dashboard/src/components/widget/button-dropdown/che-button-dropdown.styl]: yes it is !

**Comment by slemeur** [comment]
+1 from me. Changes have mostly been reviewed over calls.

File: dashboard/src/components/widget/button-dropdown/che-button-dropdown.styl (hunk 1 above)
Comment (benoitf) [inline dashboard/src/components/widget/button-dropdown/che-button-dropdown.styl]: @akurinnoy fixed

**Comment by akurinnoy** [comment]
+1

**Pull request merged**